ouroboros-network/src/Ouroboros/Network/Point.hs

```diff
@@ -19,40 +19,14 @@ module Ouroboros.Network.Point
 import           GHC.Generics (Generic)
 
 import           Cardano.Prelude (NoUnexpectedThunks)
-
+import           Cardano.Slotting.Slot
```

Comment:
Move it one line up, so that it is grouped with `Cardano.Prelude`.

Actually, the `Codec.Serialise` import should be grouped with the `GHC.Generics` import.

I don't think we have defined this as a convention, but it is the de facto convention Edsko (I believe) and I are following.